modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/streaming/DefaultStreamingHelperTestCase.java

```diff
@@ -52,7 +52,8 @@ protected void doSetUp() throws Exception {
     cursorProviderFactory =
         new InMemoryCursorIteratorProviderFactory(InMemoryCursorIteratorConfig.getDefault(), streamingManager);
     event = testEvent();
-    streamingHelper = new DefaultStreamingHelper(cursorProviderFactory, streamingManager, event);
+    streamingHelper =
+        new DefaultStreamingHelper(cursorProviderFactory, streamingManager, event, event.getContext().getOriginatingLocation());
```

Comment:
i know this is a test, but this may be confusing if looking for examples of this. Can you pass a mock or new location for this?